src/serve.rs

```diff
@@ -113,20 +114,24 @@ async fn channel(
         let first_msg = messages.next().unwrap();
 
         html! {
-            li.username {
-                span.avatar {
-                    img alt="" src=(&first_msg.avatar) {}
+            div.msg-container {
```

Comment:
Had to wrap username + messages in a container to make it possible to trigger hover/click for the whole thing